webplugin/js/app/kommunicate-ui.js

```
@@ -1538,6 +1542,33 @@ KommunicateUI = {
                 'km-animate',
                 'n-vis'
             );
+            var el = document.querySelector('.chat-popup-widget-text-wrapper');
+            if (el && window.parent && window.parent !== window) {
+                var rect = el.getBoundingClientRect();
+                if (popupTemplateKey === KommunicateConstants.CHAT_POPUP_TEMPLATE.HORIZONTAL) {
+                    var textEl = el.querySelector('.chat-popup-widget-text');
+                    var width = el.getBoundingClientRect().width;
+
+                    if (textEl) {
+                        var rect = textEl.getBoundingClientRect();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'rect'. Consider to delete it or rename it.

'rect' is defined here:


(lint/suspicious/noRedeclare)